tox.ini

```diff
@@ -68,10 +68,10 @@ basepython = python3
 commands =
     functest-run-suite --test-directory {toxinidir}/tests/functional {posargs}
 
-[testenv:func-debug]
+[testenv:func-target]
 basepython = python3
 commands =
-    functest-run-suite --test-directory {toxinidir}/tests/functional {posargs} --keep-model --log DEBUG
+    functest-run-suite --test-directory {toxinidir}/tests/functional --bundle {posargs}
```

Comment:
It seems that the difference between `func` and `func-target` is that `--bundle` is forced in the latter. These would run like:
* `tox -e func -- --keep-all-models --log DEBUG`: keeps all (logging in debug mode), and keeping all the Juju models
* `tox -e func -- --keep-faulty-model` : runs all and keeps the Juju models that failed
* `tox -e func`: runs all and does not keep any Juju model
* `tox -e func-target -- ceph-mon`: only runs the `ceph-mon.yaml` bundle. In this case the double hyphen (`--`) can be removed because there is no arg starting with hyphens
* `tox -e func-target -- ceph-mon --keep-all-models --log DEBUG`: runs the same bundle as before, but zaza does not destroy it. Logs are at debug level.

Reply:
If the above is correct, I think it should be documented.

Reply:
Yes, all above should work, only with one small change (`tox -e func-target -- ceph --keep-model --log DEBUG` no need to use `--keep-all-models` if only one model is deployed). I'll add this into README.md.